docs/reference/getting-started.asciidoc

```diff
@@ -31,156 +31,158 @@ trial of Elasticsearch Service] in the cloud.
-
-[float]
-[[run-elasticsearch-win]]
-=== Run {es} on Windows
-
-For Windows users, we recommend using the {ref}/windows.html[MSI Installer package]. The package contains a graphical user interface (GUI) that guides you through the installation process.
-
-First, download the Elasticsearch {version} MSI from
-https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-{version}.msi.
-
-Then double-click the downloaded file to launch the GUI. Within the first screen, select the deployment directories:
-
-[[getting-started-msi-installer-locations]]
-image::images/msi_installer/msi_installer_locations.png[]
-
-Then select whether to install as a service or start Elasticsearch manually as needed.
-To align with the Linux example, choose not to install as a service:
-
-[[getting-started-msi-installer-service]]
-image::images/msi_installer/msi_installer_no_service.png[]
-
-For configuration, simply leave the default values:
-
-[[getting-started-msi-installer-configuration]]
-image::images/msi_installer/msi_installer_configuration.png[]
-
-Again, to align with the tar example, uncheck all plugins to not install any plugins:
-
-[[getting-started-msi-installer-plugins]]
-image::images/msi_installer/msi_installer_plugins.png[]
-
-After clicking the install button, Elasticsearch will be installed:
-
-[[getting-started-msi-installer-success]]
-image::images/msi_installer/msi_installer_success.png[]
-
-By default, Elasticsearch will be installed at `%PROGRAMFILES%\Elastic\Elasticsearch`. Navigate here and go into the bin directory as follows:
-
-**with Command Prompt:**
-
-[source,sh]
++
+Windows:
++
+["source","sh",subs="attributes,callouts"]
 --------------------------------------------------
 cd %PROGRAMFILES%\Elastic\Elasticsearch\bin
+.\elasticsearch.exe
 --------------------------------------------------
++
+You now have a single-node {es} cluster up and running!
 
-**with PowerShell:**
-
-[source,powershell]
+. Start two more instances of {es} so you can see how a typical multi-node
+cluster behaves. You need to specify unique data and log paths
+for each node.
++
+Linux and macOS:
++
+["source","sh",subs="attributes,callouts"]
 --------------------------------------------------
-cd $env:PROGRAMFILES\Elastic\Elasticsearch\bin
+./elasticsearch -Epath.data=data2 -Epath.logs=log2
+./elasticsearch -Epath.data=data3 -Epath.logs=log3
 --------------------------------------------------
-
-And now we are ready to start our node and single cluster:
-
-[source,sh]
++
+Windows:
++
+["source","sh",subs="attributes,callouts"]
 --------------------------------------------------
-.\elasticsearch.exe
+.\elasticsearch.exe -Epath.data=data2 -Epath.logs=log2
+.\elasticsearch.exe -Epath.data=data3 -Epath.logs=log3
+--------------------------------------------------
++
+The default configuration enables the new nodes to automatically join the
+cluster with the first node because they are all running locally. Each node
+is automatically assigned a unique ID.
```

Comment:
This paragraph seems a little awkward.

Suggestion:

> By default, new nodes automatically join the same local cluster. {es} automatically assigns each node a unique ID.

Reply:
Fixed